.github/scripts/junit.py

```diff
@@ -144,7 +144,10 @@ def pretty_time_duration(seconds: float) -> str:
                         required=False,
                         default="build/junit-xml/**/*.xml",
                         help="Path to XML files. Glob patterns are supported.")
-
+    parser.add_argument("--done-file",
+                        required=False,
+                        default="",
```

Comment:
out of curiosity, why not set default value like `build/junit-xml/done`?

Reply:
I wanted this to be an optional argument so its easier to test locally.

Reply:
The "--path" has default value, so that is why I raise this comment 😃 

For another trivial reason, we can simplify the command passed to junit parser if it has default value.